src/test/kotlin/org/coepi/api/base/LocalDynamoDB.kt

```diff
@@ -0,0 +1,61 @@
+package org.coepi.api.base
+
+import com.amazonaws.AmazonServiceException
+import com.amazonaws.client.builder.AwsClientBuilder
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDBClientBuilder
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBMapper
+import com.amazonaws.services.dynamodbv2.document.DynamoDB
+import com.amazonaws.services.dynamodbv2.local.main.ServerRunner
+import com.amazonaws.services.dynamodbv2.model.*
+import org.coepi.api.dao.ReportsDaoTest
+
+class LocalDynamoDB {
+
+    fun startDynamoDb(){
+        val localUrl= "http://localhost:9000";
+        val region = "us-west-2"
+
+        println("Trying to start dynamo db at $localUrl")
+        System.setProperty("sqlite4java.library.path", "./build/libs")
+        val server = ServerRunner.createServerFromCommandLineArgs(arrayOf("-inMemory", "-port", "9000"))
+        server.start();
+
+        val ddb = AmazonDynamoDBClientBuilder.standard()
+                .withEndpointConfiguration(AwsClientBuilder.EndpointConfiguration(localUrl, region))
+                .build();
+        var ddbMapper = DynamoDBMapper(ddb);
+        createTable(ddb, ddbMapper)
+
+    }
+
+    /**
+     * TODO: Make it generic to create table based on class name passed to this method.
+     */
+    private fun createTable(ddb: AmazonDynamoDB, ddbMapper: DynamoDBMapper){
+        val request = ddbMapper.generateCreateTableRequest(ReportsTableTest::class.java)
+                .withProvisionedThroughput(ProvisionedThroughput(10, 10))
+        try{
+            ddb.createTable(request)
+            println("Created Reports table for test")
+            //fireTestQuery(ddb) // TODO: Invoke helper method to check the vales in the initialized table
+        } catch (ex: AmazonServiceException){
+            println(ex.errorMessage)
```

Comment:
this should be thrown

Reply:
Do you mean after catching the exception just throw it again??

Reply:
I mean why swallow the exception at all ? If the table creation fails, unit tests can't proceed.

Reply:
Ok. I can remove the try catch.

Reply:
Keeping to TODO comments as is to address in future PRs.